docs/authentication/saml/azure.mdx

```diff
@@ -17,77 +17,83 @@ description: Learn how to allow users to sign into your Clerk app with their Mic
-  1. Select the check box next to the user or group that you want to assign.
+  1. In the **Getting Started** section, select the **Assign users and groups** link.
+  1. Select **Add user/group**. You will be redirected to the **Add Assignment** page.
+  1. Select the **None Selected** link.
+  1. To assign a user to the enterprise application, you can either use the search field to find a user or select the checkbox next to the user in the table.
   1. Select the **Select** button at the bottom of the page.
-  1. Select the **Assign** button at the bottom of the page.
+  1. You will be redirected to the **Add Assignment** page. Select the **Assign** button at the bottom of the page.
 
-  ### Configure single sign-on (SSO) in Microsoft
+  ### Configure SSO in Microsoft
 
-  After you have assigned the user or group of users to the enterprise application, you need to configure the single sign-on (SSO) settings to enable SAML SSO.
+  After assigning the user or group to the enterprise application, you need to configure the SSO settings to enable SAML SSO.
 
-  1. In the navigation sidebar, select **Manage** and select **Single sign-on**.
-  1. Select **SAML** as the single sign-on method.
+  1. In the navigation sidebar, open the **Manage** dropdown and select **Single sign-on**.
+  1. In the **Select a single sign-on method** section, select **SAML**.
 
   ### Configure your service provider
 
-  To configure your service provider (Clerk), you must add these two fields to your IdP's application:
+  To configure Clerk as your service provider, add these two fields to your IdP's application:
 
-  - **Reply URL (Assertion Consumer Service URL)** - This is your application's URL that your IdP will redirect your users back to after they have authenticated in your IdP.
-  - **Identifier (Entity ID)** - This is a unique identifier for your SAML connection that your IdP application needs.
+  - **Identifier (Entity ID)** - A unique identifier for your SAML connection, required by your IdP application.
+  - **Reply URL (Assertion Consumer Service URL)** - The URL of your application where your IdP will redirect your users after they authenticate.
 
   To fill out the appropriate values for these fields:
 
   1. On the **Set up Single Sign-On with SAML** page, find the **Basic SAML Configuration** section.
-  1. Select the **Edit** button. This will open a **Basic SAML Configuration** modal.
-  1. Paste the **Reply URL (Assertion Consumer Service URL)** and the **Identifier (Entity ID)** values that you saved earlier into their respective fields.
-  1. Select **Save** at the top of the modal.
+  1. Select **Edit** to open the **Basic SAML Configuration** panel.
+  1. Paste the **Identifier (Entity ID)** and **Reply URL (Assertion Consumer Service URL)** values you saved from the previous step into their respective fields. These values will be saved automatically.
+  1. Select **Save** at the top of the panel.
```

Comment:
I see you added "These values will be saved automatically" to the step before this one. if they're saved automatically, why does the user need to select save